core/internal/cltest/README.md

```diff
@@ -0,0 +1,3 @@
+Importing cltest should only be for top level black box (X_test package) integration tests, 
+for example testing against the Application object itself. 
+Consider using a smaller scoped package: utils or testutils/* if possible. 
```

Comment:
nit/ could be a package level comment in a `doc.go` so it shows on https://pkg.go.dev/github.com/smartcontractkit/chainlink@v1.6.0/core/internal/cltest etc.?
```suggestion
// Importing cltest should only be for top level black box (X_test package) integration tests, 
// for example testing against the Application object itself. 
// Consider using a smaller scoped package: utils or testutils/* if possible. 
package cltest
```